java/src/jmri/jmrit/roster/Roster.java

```diff
@@ -1232,6 +1233,9 @@ public void copyRosterGroupList(String oldName, String newName) {
             re.putAttribute(newGroup, "yes"); // NOI18N
         });
         this.addRosterGroup(new RosterGroup(newName));
+        // would expect the firePropertyChange event to be called by addRosterGroup()
+        // but has to be repeated here for the Group list to redraw:
+        firePropertyChange(ROSTER_GROUP_ADDED, null, newName);
```

Comment:
That whatever is listening for this property to change does not update itself on the change is a bug in that listener, not in the Roster and should be fixed by fixing the listener, not by duplicating the message, since now all listeners get the message twice (which can cause a lot of network traffic if, for example, web throttles are in use).